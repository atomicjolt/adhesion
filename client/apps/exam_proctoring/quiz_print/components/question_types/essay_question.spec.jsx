import React from 'react';
import { shallow } from 'enzyme';
import Essay from './essay_question';

describe('Essay', () => {
  let result;
  let props;

  beforeEach(() => {
    props = {
      question_text: '<p>What is love?</p>',
      id: '7',
    };
    result = shallow(<Essay {...props} />);
  });

  it('matches the snapshot', () => {
    expect(result).toMatchSnapshot();
  });
});
