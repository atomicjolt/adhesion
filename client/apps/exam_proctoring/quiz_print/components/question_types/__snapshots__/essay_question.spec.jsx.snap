// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Essay matches the snapshot 1`] = `ShallowWrapper {}`;
